src/views/organisationType.njk

```diff
@@ -1,5 +1,58 @@
 {% extends 'base.njk' %}
 {% set title = 'Organisation Type' %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block beforeContent %}
+
+    {{ govukBackLink({
```

Comment:
If you remove the back link component to the next block down `{% block content %}` this allows the back button appears on the page
Then you can remove the `before block` as anything in this block will be hidden by default... that'll add your back routing :)

Reply:
I included the `{block beforeContent`} based on the guidance from GDS [https://design-system.service.gov.uk/components/back-link/](url) which recommends placing back links before the <main> element. The only way for me to achieve this without modifying the base.njk file too much ,add the` block beforeContent` to the base.njk file and then add the link in the form page files. If this detail isn't crucial for the demo, I can follow your suggestion and add the back link directly to the `{block content}`. Alternatively instead of `{block beforeContent}`, I could include the link as part of the base template file before the <main> tag, but that would require adding logic to ensure it doesn't render on home page for example

Reply:
Ohhh ok i can see what you mean now, ok it wasn't appearing for me before but now i realise what you mean... ok that makes sense i'll resolve this comment!